source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRUIImageView.mm

```diff
@@ -6,10 +6,28 @@
 //
 
 #import "ACRUIImageView.h"
+#import "ACRContentHoldingUIView.h"
 
 @implementation ACRUIImageView
 
--(void)layoutSubviews
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    if(self) {
+        self.tag = eACRUIImageTag;
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if(self) {
+        self.tag = eACRUIImageTag;
+    }
+    self.desiredSize = frame.size;
```

Comment:
I don't see why one of these statements is inside the `if` block and the other isn't.